### PR TITLE
vdk-postgres: batch inserts during ingestion

### PR DESCRIPTION
**Pull request by antoniivanov**
Insert data is painfully slow even small amounts making even testing and pocs hard. So this is a quick and slight optimization Before insert 300 rows took 15 minutes. Now it's about 1 minute

Testing Done: made sure existing test pass. Tested with a job sending 300 rows in 2 tables.

### CODE REVIEW

File: projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py [hunk 1]
```diff
@@ -57,28 +58,19 @@ def ingest_payload(
...
-        'INSERT INTO dest_table (val1, val2) VALUES (%s, %s), (%s, %s), (%s, %s)', ['val1', 'val2']
-
-        :param destination_table: str
-            the name of the destination table
-        :param cursor: PEP249Cursor
-            the database cursor
-        :param payload: List[dict]
-            the payloads to be ingested
-        :return: Tuple[str, Tuple[str]]
-            tuple containing the query and parameters
```
Comment (gageorgiev) [inline projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py:73]: why delete the docstring?

**Review by gageorgiev — APPROVED** [review]
would love to see a before/after on ingesting the same confluence pages we discussed, so I have an idea as to how much of an improvement this is with respect to our POC

**Review by murphp15 — APPROVED** [review]

File: projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py [hunk 2]
```diff
@@ -57,28 +58,19 @@ def ingest_payload(
...
+        Prepare the SQL query and parameters for bulk insertion.
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py:63]:
```suggestion
        Prepare the SQL query and parameters for bulk insertion.
        
        Returns insert into destination table tuple of query and parameters;
        E.g. for a table dest_table with columns val1, val2 and payload size 2, this method will return:
        'INSERT INTO dest_table (val1, val2) VALUES (%s, %s)',
        [('val1', 'val2'), ('val1', 'val2')]
```

File: projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py (hunk 1 above)
Comment (antoniivanov) [reply projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py:73]: The param comments are redundant.

 But the description was good. I returned it (after adjusting it to the new interface).

**Comment by antoniivanov** [comment]
> would love to see a before/after on ingesting the same confluence pages we discussed, so I have an idea as to how much of an improvement this is with respect to our POC


@gageorgiev . It's in the description :
> Before insert 300 rows took 15 minutes. Now it's about 1 minute

**Pull request merged**